frontend/src/routes/_protected/treecluster/index.tsx

```diff
@@ -33,6 +33,7 @@ function Treecluster() {
 
   const [statusTags, setStatusTags] = useState<string[]>(search.status);
   const [regionTags, setRegionTags] = useState<string[]>(search.region);
+  const [searchQuery, setSearchQuery] = useState<string>('');
```

Comment:
I don't want to be the person who complains about the naming of a variable, but here I thought you were trying something with the query search parameter, as I notice its the actual string from the search input, so maybe we can choose a different name for tis state?